src/main/java/net/dv8tion/jda/api/entities/Invite.java

```diff
@@ -446,6 +446,15 @@ default String getUrl()
          */
         @Nonnull
         Set<String> getFeatures();
+
+        /**
+         * The welcome screen of the {@link net.dv8tion.jda.api.entities.Invite.Guild Guild}.
+         * <br>This will be {@code null} if the Guild has no welcome screen.
+         *
+         * @return The welcome screen of this Guild or {@code null}
+         */
+        @Nullable
+        GuildWelcomeScreen getWelcomeScreen();
```

Comment:
So all resolved invites have this information, but the Guild itself doesnt? Curious

Reply:
exactly, you have to retrieve it yourself.